composer.json

```diff
@@ -26,11 +26,8 @@
     "jcalderonzumba/gastonjs": "~1.0"
   },
   "require-dev": {
-    "symfony/process": "~2.3",
-    "symfony/phpunit-bridge": "~2.7",
-    "symfony/css-selector": "~2.1",
-    "phpunit/phpunit": "~4.6",
-    "silex/silex": "~1.2"
```

Comment:
Silex and Process are left-over from the split of gastonjs (which is the one needing them for its tests). CssSelector is a Mink dependency (and your requirement was forbidding to use latest version of Symfony).
And the PHPUnit Bridge is enforced by the new mink/driver-testsuite repo (and also allowing newer versions of the package)